run_pseudobulk_benchmark.py

```diff
@@ -147,6 +148,7 @@
         adata_train=adata_train,
         adata_pseudobulk_test_counts=adata_pseudobulk_test_counts,
         adata_pseudobulk_test_rc=adata_pseudobulk_test_rc,
+        all_adata_samples_test=all_adata_samples_test,
```

Comment:
To make sure I follow - with the new `get_latent_representation` function we use, we only need `all_adata_samples` at inference, the latent pseudobulk is computed inside. However we still need `adata_pseudobulk_test_rc` for the baselines (nnls, TAPE etc ...) and `adata_pseudobulk_test_counts` for DestVI.

Lmk if that makes sens.

Reply:
Did you check that this change did not impact other baselines btw ?

Reply:
Yes exactly! 
Normally it shouldn't affect other baselines, since it is explicit in sanity_check (I think) where there is `elif model == "MixUpVI"`